packages/react-cs-renderer/src/ReactNativeCS.js

```diff
@@ -227,4 +247,48 @@ injectInternals({
   rendererPackageName: 'react-cs-renderer',
 });
 
-module.exports = ReactNativeCSFiber;
+type ReactCSProps = {children: ReactNodeList};
+type ReactCSState = {
+  root: Object,
+  container: {
+    pendingChild: null | Instance | TextInstance,
+  },
+};
+
+const ReactCS = CSStatefulComponent({
+  getInitialState({props: ReactCSProps}): ReactCSState {
+    let container = {
+      pendingChild: null,
+    };
+    let root = ReactNativeCSFiberRenderer.createContainer(container, false);
+    return {root, container};
+  },
+  render({
+    props,
+    state,
+    stateUpdater,
+  }: {
+    props: ReactCSProps,
+    state: ReactCSState,
+    stateUpdater: (update: (oldState: ReactCSState) => ReactCSState) => void,
+  }) {
+    ReactNativeCSFiberRenderer.updateContainer(
+      props.children,
+      state.root,
+      null,
+      null,
+    );
+    if (scheduledCallback) {
+      const callback = scheduledCallback;
+      scheduledCallback = null;
+      callback(infiniteDeadline);
+    }
```

Comment:
@acdlite This is basically a hack around the fact that we can't just call performWork on a per root basis manually.